src/wallet/wallet2.cpp

```diff
@@ -6602,6 +6617,11 @@ bool wallet2::parse_tx_from_str(const std::string &signed_tx_st, std::vector<too
       LOG_PRINT_L0("Failed to decrypt signed transaction: " << e.what());
       return false;
     }
+    catch(...)
+    {
+    LOG_PRINT_L0("Failed to parse decrypted data from signed transaction");
```

Comment:
Likewise shouldn't this be "Failed to decrypt signed transaction"?

Reply:
correct